server/scripts/feed-sites.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-console */
+require('dotenv').config({ path: '../.env' });
+const readXlsxFile = require('node-xlsx').default;
+const path = require('path');
+const { dbClient } = require('../db/db.js');
+const { saveSites } = require('../services/employers.js');
+const { createRows, verifyHeaders } = require('../utils');
+
+(async () => {
+  if (require.main === module) {
+    try {
+      const columnMap = {
+        'HCAP Site ID': 'siteId',
+        'Site Name': 'siteName',
+        'Early Adopter Allocation': 'earlyAdopterAllocation',
+        'Street Address': 'address',
+        'Health Authority': 'healthAuthority',
+        City: 'city',
+        'Post Code': 'postalCode',
+        'Registered Business Name': 'registeredBusinessName',
+        'Operator Contact First Name': 'operatorContactFirstName',
```

Comment:
There's a missing field here, Operator name. This is actually a separate field to first/last since it might be a company name and not a person :)